spec/chewy/type/adapter/active_record_spec.rb

```diff
@@ -63,7 +65,10 @@
   describe '#import' do
     def import(*args)
       result = []
-      subject.import(*args) { |data| result.push data }
+      subject.import(*args) do |data|
+        result.push data
+        yield if block_given?
+      end
       result
     end
```

Comment:
An alternative solution. A bit too fancy because it uses the fact that `each` returns the enumerable as array if block is given. If block is not given, it returns a lazy enumerable. This is why we need `.to_a` at the end.

```ruby
def import(*args, &block)
  subject.to_enum(:each, *args).each(&block).to_a
end
```